packages/libs/src/services/dataContracts/EntityManagerClient.ts

```diff
@@ -31,7 +31,13 @@ export enum Action {
   PIN = 'Pin',
   UNPIN = 'Unpin',
   MUTE = 'Mute',
-  UNMUTE = 'Unmute'
+  UNMUTE = 'Unmute',
+  ADD_EMAIL = 'AddEmail',
+  UPDATE_EMAIL = 'UpdateEmail',
+  ADD_EMAIL_KEY = 'AddEmailKey',
+  UPDATE_EMAIL_KEY = 'UpdateEmailKey',
+  ADD_ACCESS_KEY = 'AddAccessKey',
+  UPDATE_ACCESS_KEY = 'UpdateAccessKey'
```

Comment:
Our upload saga currently uses `@audius/sdk`'s exported `EntityManagerActions` but then calls into libs (though not for long!), and having the two out of sync means it'll type error. Added this to keep them in sync for now, though #10541 is imminent anyway